server/tests/unit/controllers/authentication_controller.ts

```diff
@@ -0,0 +1,160 @@
+import { expect } from "chai";
+import { createSandbox, SinonStub } from "sinon";
+import { AuthenticationController } from "../../../src/controllers/authentication_controller";
+import { Gender } from "../../../src/entities/gender";
+
+describe("authentication_controller.ts", () => {
+    const authenticationService: any = {
+        signUp: Function,
+        signIn: Function,
+    };
+    const controller = new AuthenticationController(authenticationService);
+
+    const sandbox = createSandbox();
+
+    let req: any;
+    let res: any;
+    let resJsonStub: SinonStub;
+
+    beforeEach(() => {
+        res = {
+            json: Function,
+        };
+
+        sandbox.restore();
+        resJsonStub = sandbox.stub(res, "json");
+    });
+
+    describe("AuthenticationController::signUp", () => {
```

Comment:
I should have added this on my PR, thx!
will add unit test for every end point next time.